roles/pulp_webserver/tasks/apache.yml

```diff
@@ -57,7 +57,7 @@
     # which is a color ANSI color code.
     - name: Symlink Apache snippets
       file:
-        src: "{{ __pulp_webserver_snippet.stdout_lines | first }}"
+        src: "{{ __pulp_webserver_snippet.stdout | trim | regex_replace('\\s+.*$', '') }}"
```

Comment:
So we put "first" in here because sometimes the command that produces __pulp_webserver_snippet adds a line at the end. stderr for an interpeter error or something I think.

Maybe we need both "first" and "trim"?

Reply:
```
TASK [pulp.pulp_installer.pulp_webserver : Symlink nginx snippets] *************
changed: [pulp3-source-centos8-fips] => (item={'changed': False, 'rc': 0, 'stdout': '/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf\r\n\x1b[0m', 'stdout_lines': ['/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf', '\x1b[0m'], 'stderr': 'Shared connection to 192.168.122.94 closed.\r\n', 'stderr_lines': ['Shared connection to 192.168.122.94 closed.'], 'failed': False, 'failed_when_result': False, '__pulp_webserver_plugin': {'key': 'pulp-ansible', 'value': {'source_dir': '/home/vagrant/devel/pulp_ansible'}}, 'ansible_loop_var': '__pulp_webserver_plugin'})
changed: [pulp3-source-centos8-fips] => (item={'changed': False, 'rc': 0, 'stdout': '/home/vagrant/devel/pulp_container/pulp_container/app/webserver_snippets/nginx.conf\r\n\x1b[0m', 'stdout_lines': ['/home/vagrant/devel/pulp_container/pulp_container/app/webserver_snippets/nginx.conf', '\x1b[0m'], 'stderr': 'Shared connection to 192.168.122.94 closed.\r\n', 'stderr_lines': ['Shared connection to 192.168.122.94 closed.'], 'failed': False, 'failed_when_result': False, '__pulp_webserver_plugin': {'key': 'pulp-container', 'value': {'source_dir': '/home/vagrant/devel/pulp_container'}}, 'ansible_loop_var': '__pulp_webserver_plugin'})

```

Reply:
I added a debug to check:
```
- name: Debug
      debug:
        msg: "{{ __pulp_webserver_snippet.stdout | trim | regex_replace('\\s+.*$', '') }}"
      loop: '{{ snippets.results }}'
      loop_control:
        loop_var: __pulp_webserver_snippet
      when: __pulp_webserver_snippet.rc == 0
```
and it seems to remove the ANSI color code:
```
TASK [pulp.pulp_installer.pulp_webserver : Debug] ******************************
ok: [pulp3-source-centos8-fips] => (item={'changed': False, 'rc': 0, 'stdout': '/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf\r\n\x1b[0m', 'stdout_lines': ['/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf', '\x1b[0m'], 'stderr': 'Shared connection to 192.168.122.94 closed.\r\n', 'stderr_lines': ['Shared connection to 192.168.122.94 closed.'], 'failed': False, 'failed_when_result': False, '__pulp_webserver_plugin': {'key': 'pulp-ansible', 'value': {'source_dir': '/home/vagrant/devel/pulp_ansible'}}, 'ansible_loop_var': '__pulp_webserver_plugin'}) => {
    "__pulp_webserver_snippet": {
        "__pulp_webserver_plugin": {
            "key": "pulp-ansible",
            "value": {
                "source_dir": "/home/vagrant/devel/pulp_ansible"
            }
        },
        "ansible_loop_var": "__pulp_webserver_plugin",
        "changed": false,
        "failed": false,
        "failed_when_result": false,
        "rc": 0,
        "stderr": "Shared connection to 192.168.122.94 closed.\r\n",
        "stderr_lines": [
            "Shared connection to 192.168.122.94 closed."
        ],
        "stdout": "/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf\r\n\u001b[0m",
        "stdout_lines": [
            "/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf",
            "\u001b[0m"
        ]
    },
    "msg": "/home/vagrant/devel/pulp_ansible/pulp_ansible/app/webserver_snippets/nginx.conf"
}
```

Reply:
Update the comment to say that we solve for bad output like:
stdout_lines": ["", "/usr/lib/python3.8/site-packages/pulp_container/app/webserver_snippets/nginx.conf", "\u001b[0m"]

Reply:
I'll add a comment in a follow-up PR